### DISCUSSION TITLE
Power relais cycle by esp32 but not by 8266 devices?

### DISCUSSION THREAD

**Discussion by CosmicDuck** [arendst/Tasmota#22631] [Support request] [answered]
Hello,

I have a bunch of older different 8266 based tasmota devices an now 4 newer esp32 based smart plug devices.
All they came with tasmota allready installed.

I wonder me but if a reboot is needed all my 8266 based devices will hold the power relais in there state as is.
Against the esp32 based devices produce a short power cycle if the relais is in on state so the device behind the smart plug is also restarted (which is not helpfull, my server does not like it).

Is this a normal behavior of the 8266 and esp32 devices (8266 never cycle and esp cycle at reboot)?

**Comment by arendst** [comment]
Pls provide info like asked on opening an issue.  Most important this time is type of esp32 and tasmota version.

**Reply by CosmicDuck** [reply]
Thank you. I though maybe this is a general issue of esp32 devices which cannot be changed and most user accept this behavior as normal. By the way maybe this is a issue of hardware so I will write below in the other anwer.

**Comment by Noschvie** [comment]
Have a look to "ESP32 + Relay + integrated 230V AC Power Supply" #21056

**Reply by CosmicDuck** [reply]
Hello, thank you for the hint. By the way the EIGHTREE device looks like to use the GPIO6 for Relay. Is there a special meaning on this GPIO?. If you search for ASIN B0CHMMKZCQ you can find the device.
Here a little bit more Infos:
```
EIGHTREE
MeinSachs2WCHeizschalter
Program Version	14.3.0 (release-tasmota32)
Build Date & Time	2024-10-15T08:21:39
Core/SDK Version	3_1_0/5.3.1.240924
Uptime	15T01:10:23
Flash Write Count	1851
Boot Count	141
Restart Reason	Vbat power on reset
…
HTTP API	Enabled
MQTT Host	192.168.2.xxx
MQTT Port	1883
MQTT TLS	Disabled
MQTT User	mqtt
MQTT Client	meinsachs2wcheizschalter
MQTT Topic	MeinSachs2WCHeizschalter
MQTT Group Topic 1	cmnd/tasmotas/
MQTT Full Topic	cmnd/MeinSachs2WCHeizschalter/
MQTT Fallback Topic	cmnd/meinsachs2wcheizschalter_fb/
MQTT No Retain	Disabled
Emulation	None
mDNS Discovery	Disabled
ESP Chip Id	10813452 (ESP8685 v0.4)
Flash Chip Id	0x164020 (DIO)
Flash Size	4096 KB
Program Flash Size	4096 KB
Program Size	1997 KB
Free Program Space	882 KB
Free Memory	174.7 KB (frag. 36%)
Partition safeboot	832 KB (used 86%)
Partition app0*	2880 KB (used 69%)
Partition fs	320 KB
```

![grafik](https://github.com/user-attachments/assets/698995aa-dedc-4b23-bced-7ac89db112d4)

**Reply by Jason2866** [reply]
Please try latest release. There are changes which may solve.

**Reply by CosmicDuck** [reply]
I found out for the esp32c3 the gpio6 could be have a special meaning becaus it could be used for jtag/spi2?
![grafik](https://github.com/user-attachments/assets/bfebc590-a147-489f-8061-62fcdccb3925)

**Reply by CosmicDuck** [reply, marked as answer]
> Please try latest release. There are changes which may solve.
Yea, it looks like has to been fixed in 14.4. I mean I updated yesterday to 14.3 so the problem is gone today.

**Reply by CosmicDuck** [reply]
It would be nice if someone can add the closed issue link. I cannot not find it maybe it is fixed without an opened issue.

**Reply by Jason2866** [reply]
No issue. Esp32 Relais blinking when restarted was an enhancement
